packages/grid/x-data-grid/src/tests/filterPanel.DataGrid.test.tsx

```diff
@@ -471,9 +471,8 @@ describe('<DataGrid /> - Filter panel', () => {
     expect(onFilterModelChange.lastCall.args[0].items[0].value).to.equal(undefined);
 
     deleteFilterForm();
-    await waitFor(() => {
-      expect(screen.queryAllByRole('tooltip').length).to.deep.equal(0);
-    });
+    clock.tick(100);
```

Comment:
`clock.tick(100)` might not always be enough depending on the. environment.
Isn't `await waitFor` less flaky?